network/p2p/inspector/validation/control_message_validation_inspector_test.go

```diff
@@ -15,13 +15,70 @@ import (
 	"github.com/onflow/flow-go/module/irrecoverable"
```

Comment:
Observed that the package name is `validation` rather than `validation_test`, risking import cycles between test and code packages. If feasible, please rename the package to avoid this issue.

Reply:
https://github.com/onflow/flow-go/pull/4773/commits/265d00ecafd0eab04e6db535f231417ba6564c87